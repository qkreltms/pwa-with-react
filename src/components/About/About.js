import React from 'react'
import styles from './About.css'

const About = () => {
  return (
    <div>
      <h1 className={styles.heading}>Hi. I am JungHoon!</h1>
      <p className={styles.content}>This section is all about me, yay!</p>
    </div>
  )
}

export default About
