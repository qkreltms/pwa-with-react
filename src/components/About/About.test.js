/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import About from './About'
import toJson from 'enzyme-to-json'

test('About component should render as expected', () => {
  const component = shallow(<About />) // About Dom을 가져옴  
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
