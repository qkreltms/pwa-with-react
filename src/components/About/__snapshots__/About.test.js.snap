// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render as expected 1`] = `
<div>
  <h1
    className="heading"
  >
    Hi. I am JungHoon!
  </h1>
  <p
    className="content"
  >
    This section is all about me, yay!
  </p>
</div>
`;
